Online-VS-Offline-IDE.md

```diff
@@ -0,0 +1,38 @@
+# Online vs Offline IDEs
+
+## Offline IDEs
+
+There are several offline [IDE](Integrated Development Environment) that programmers use in order to stay more productive to learn and be more productive at work.
+
+- **[Eclipse](https://www.app5.unisys.com/offerings/ClearPathConnection/eclipse.html)** is more than a Java editor. Its biggest advantages are code completion by tabbing method which saves a lot of time when writing documentation.
+
+  It has a build in syntax check to fix any mistype words while coding a project. All of features we've come to expect of an IDE are their code completion, templates, integration  with various SCMSs, and integration with build systems.
+
+  Its code formatting and cleanup tools are very powerful. I find its build system to work well and intuitively. I think these are the things upon which its reputation is really built.
+
+  Furthermore it has a refactoring that find and replace function, variables and classes of a specific work.
+
+  Eclipse offers more plugins, platforms, frameworks, and is simpler to use compare  to other IDEs such as **IntelliJ**, **BlueJ** and **JEdit**.
```

Comment:
BlueJ is the simplest of IDEs and used to taught Java to teenage kids. Also, I don't think Eclipse is simpler or offers more than IntelliJ Idea.